src/app/modules/workers/pages/workers-page/workers-facade.service.ts

```diff
@@ -19,8 +23,44 @@ export class WorkersFacadeService {
     });
   }
 
+  addWorker(worker: Employee) {
+    const parsedWorker = this.getTruthyValues(worker);
+    if (this.isValidWorker(parsedWorker)) {
+      this.workersClient.post(worker).subscribe((createdWorker) => {
+        this.workersService.add(createdWorker);
```

Comment:
Zamiast manipulowac stan appki i probowac synchronizowac go z prawdziwym stanem w bazie, lepiej jest po prostu pobrać na nowo workers z backendu. Inaczej nigdy nie bedziemy mieli pewnosci ze obecny stan odzwierciedla to co mamy na bazie, bo mogło przecież coś przy zapisie do bazy pójść nie tak